geps/gep-1713.md

```diff
@@ -0,0 +1,220 @@
+
+
+Failure to attach a "child" Gateway to another MUST result in the "child" Gateway setting their `Accepted` condition to `False`. This MUST happen in the following scenarios:
+- Attaching to another "child" Gateway
+- Attaching to oneself (self-referential)
+- Attaching to a non-existing Gateway
+- Attaching to a Gateway with a different `GatewayClassName` 
+
+If a Gateway's `Accepted` condition is `False` then the conditions 'Programmed' and 'Ready' (if supported) MUST be set to `False`.
+
+#### Route Attaching
+
+Routes MUST be able to specify a "child" Gateways as a `parentRef` and make use of the fields in `ParentReference` to help target a specific listener.
+
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: httproute-example
+spec:
+  parentRefs:
+  - name: acme-lb
+    sectionName: https
+```
+
+#### `parent-gateway` label
+
+Implementations MUST label "child" Gateways with the `gateway.networking.k8s.io/parent-gateway` label. The value of the label is the name of the "parent" Gateway and it MUST be equivalent to the value `spec.infrastructure.attachTo.name`
+
+Users MUST NOT be able to edit this field.
+
+#### Merging Spec Fields
+
+
+#### GatewayClassName 
+
+The `GatewayClassName` MUST be set on "child" and "primary" Gateways and be equivalent.
+
+#### Listeners
+
+Implementations MUST treat the "primary" Gateway as having the concatenated list of all listeners from "child" Gateways.
+
+Validation of this list of listeners is subject to the discussion here: https://github.com/kubernetes-sigs/gateway-api/issues/1842
+
+eg.
+```yaml
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: parent-gateway
+spec:
+  gatewayClassName: example
+  listeners:
+  - name: HTTP
+    port: 80
+    protocol: HTTP 
+---
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: child-gateway
+  labels:
+    gateway.networking.k8s.io/parent-gateway: parent-gateway
+spec:
+  gatewayClassName: example
+  infrastructure:
+    attachTo:
+      name: parent-gateway
+      kind: Gateway
+      group: gateway.networking.k8s.io
+  listeners:
+  - name: metrics
+    port: 8080
+    protocol: HTTP    
+```
+
+With this configuration, the realized "primary" Gateway should listen on port `80` and `8080`.
+
+
+#### Addresses
+
+Merging `Addresses` is not supported and the "child" Gateway's `Addresses` MUST be empty.
+
+
+### Status Fields
+
+
+#### Addresses
+
+The list of `Addresses` that appear in the status of the "child" Gateway MUST be the same as the "parent" Gateway.
+
+#### Listeners
+
+The "child" Gateways MUST provide a status for each Listener defined in their specification. Listeners defined in "sibling" and the "parent" Gateway SHOULD NOT appear on a "child" Gateway.
```

Comment:
Do child listeners appear in the parent?

Reply:
I don't think it should. Otherwise edits on the children will mean the parent will need to be updated which I want to avoid

Reply:
Agreed. Good to specify this in the doc though.